fsspec/implementations/memory.py

```diff
@@ -164,6 +164,20 @@ def _rm(self, path):
         else:
             raise FileNotFoundError
 
+    def rm(self, path, recursive=False, maxdepth=None):
```

Comment:
Aside from the branch that should never get called, I don't see how this is different from `AbstractFileSystem.rm`.